Samples/Samples/View/CalendarEventPage.xaml

```diff
@@ -62,9 +62,36 @@
                     <ColumnDefinition Width="Auto"/>
                     <ColumnDefinition/>
                 </Grid.ColumnDefinitions>
-                <Label Text="Description:"/>
-                <Label Grid.Column="1"
-                   Text="{Binding Description}"/>
+              <Grid.RowDefinitions>
```

Comment:
I think the first RowDefinition should be Auto height

Reply:
Yep, I think I was just unsure how this would react if we had like 20 attendees, could it potentially push description off the page? Often this will not be the case, but was just a bit wary, perhaps I could use a maxheight property?

Reply:
put the whole thing (ie all details of the appointment) in a scroll container and that way if there's a long attendee list, the UI will scroll.
alternatively, you could put a max height on the attendee stackLayout and let it scroll. For this it might be better to put in a ListView rather than a StackLayout.

Reply:
Ironically it was a ListView previously, but Mikolaj suggested I try using BindableLayout, which is why it is a stack layout at present.

Reply:
Moved the entire grid within a scroll view as a solution for now.